app/assets/javascripts/ideal-image-slider/ideal-image-slider.min.js

```diff
@@ -0,0 +1,2 @@
xtSlide),g(this._attributes.previousSlide,this.settings.classes.previousSlide),g(this._attributes.currentSlide,this.settings.classes.currentSlide),g(this._attributes.nextSlide,this.settings.classes.nextSlide),this._attributes.currentSlide.setAttribute("aria-hidden","false"),g(this._attributes.container,this.settings.classes.directionNext),b(function(){h(this._attributes.container,this.settings.classes.directionNext)}.bind(this),this.settings.transitionDuration),this.settings.transitionDuration&&(g(this._attributes.container,this.settings.classes.animating),b(function(){h(this._attributes.container,this.settings.classes.animating)}.bind(this),this.settings.transitionDuration)),p(this),this.settings.afterChange.apply(this)},r.prototype.gotoSlide=function(a){this.settings.beforeChange.apply(this),this.stop(),h(this._attributes.previousSlide,this.settings.classes.previousSlide),h(this._attributes.currentSlide,this.settings.classes.currentSlide),h(this._attributes.nextSlide,this.settings.classes.nextSlide),this._attributes.currentSlide.setAttribute("aria-hidden","true"),a--;var c=this._attributes.slides,d=c.indexOf(this._attributes.currentSlide);this._attributes.previousSlide=c[a-1],this._attributes.currentSlide=c[a],this._attributes.nextSlide=c[a+1],"undefined"==typeof this._attributes.previousSlide&&(this._attributes.previousSlide=c[c.length-1]),"undefined"==typeof this._attributes.nextSlide&&(this._attributes.nextSlide=c[0]),k(this._attributes.previousSlide),k(this._attributes.currentSlide),k(this._attributes.nextSlide),g(this._attributes.previousSlide,this.settings.classes.previousSlide),g(this._attributes.currentSlide,this.settings.classes.currentSlide),g(this._attributes.nextSlide,this.settings.classes.nextSlide),this._attributes.currentSlide.setAttribute("aria-hidden","false"),d>a?(g(this._attributes.container,this.settings.classes.directionPrevious),b(function(){h(this._attributes.container,this.settings.classes.directionPrevious)}.bind(this),this.settings.transitionDuration)):(g(this._attributes.container,this.settings.classes.directionNext),b(function(){h(this._attributes.container,this.settings.classes.directionNext)}.bind(this),this.settings.transitionDuration)),this.settings.transitionDuration&&(g(this._attributes.container,this.settings.classes.animating),b(function(){h(this._attributes.container,this.settings.classes.animating)}.bind(this),this.settings.transitionDuration)),p(this),this.settings.afterChange.apply(this)},r.prototype.destroy=function(){clearInterval(this._attributes.timerId),this._attributes.timerId=0,this._attributes.container.innerHTML="",Array.prototype.forEach.call(this._attributes.origChildren,function(a,b){this._attributes.container.appendChild(a)}.bind(this)),h(this._attributes.container,this.settings.classes.container),h(this._attributes.container,"iis-effect-"+this.settings.effect),this._attributes.container.style.height="",this.settings.onDestroy.apply(this)},{_hasClass:f,_addClass:g,_removeClass:h,Slider:r}}();
```

Comment:
the code for this was provided by an js package called ideal-image-slider which generates a sliding image show on our home page.